test/test_plugins/test_github.py

```diff
@@ -19,16 +19,15 @@ def dicteq(a, b):
     assert sorted(a.items()) == sorted(b.items())
 
 def test_basic():
-    with vcr.use_cassette('test/fixtures/github_issues.yaml'):
-        ret = on_message({"text": u"!hub issue 5 -r llimllib/limbo", "channel": "test_channel"}, SERVER)
-        #import ipdb; ipdb.set_trace()
+    with VCR.use_cassette('test/fixtures/github_issues.yaml'):
```

Comment:
Why do this when you've deleted the fixture?

Reply:
I messed up! If I found any tests that were broken with `make test-network`, I meant to update their fixtures and then fix the tests. Instead of updating the fixtures, I deleted them 😢 

Commit a6ce207 restores these missing fixtures.